models/Enquiry.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose');
+
+const EnquirySchema = new mongoose.Schema({
+    email: {
+        type: String,
+        required: true,
+        trim: true
```

Comment:
- remove the email property
- add reference to the user

Reply:
done.